sourcemash/frontend/templates/feed-topic.tpl

```diff
@@ -0,0 +1,16 @@
+<div id="<%= topic %>" class="section scrollspy">
```

Comment:
I think scrollspy should be attached to the same ID as the href from the table of contents points to, so I'd prefer to leave it here.